sdkproject/Assets/Mapbox/Unity/MeshGeneration/Modifiers/MeshModifiers/ReplaceFeatureModifier.cs

```diff
@@ -48,42 +57,72 @@ public override void SetProperties(ModifierProperties properties)
 			_options = (SpawnPrefabOptions)properties;
 		}
 
+		public override void FeaturePreProcess(VectorFeatureUnity feature)
+		{
+			int index = -1;
+			foreach (var point in _prefabLocations)
```

Comment:
This gets really deeply nested; anyway we can avoid that?

Reply:
Check out the refactor please 😄